graylog2-web-interface/test/setup-jest.jsx

```diff
@@ -11,7 +12,16 @@ registerBuiltinStores();
 
 configure({ adapter: new Adapter() });
 
-// eslint-disable-next-line no-console
+const originalWarn = console.warn;
+
+console.warn = jest.fn((...args) => {
+  if (!args[0] || !args[0].includes('react-async-component-lifecycle-hooks')) {
+    originalWarn(format(...args));
+  }
+});
+
 console.error = jest.fn((...args) => {
-  throw new Error(format(...args));
+  if (!args[0] || !args[0].includes('react-async-component-lifecycle-hooks')) {
```

Comment:
Please do it only for warnings. We want to reduce noise, but not swallow actual errors.